docs/terraform.md

```diff
@@ -0,0 +1,18 @@
+
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-----:|:-----:|
+| flavor | Flavor depends of OS and init system | string | `debian-systemd` | no |
+| name |  | string | - | yes |
+| namespace |  | string | - | yes |
+| region | AWS region to manage instances in | string | - | yes |
+| stage |  | string | - | yes |
+
+## Outputs
+
+| Name | Description |
+|------|-------------|
+| policy_arn |  |
+| user_data |  |
```

Comment:
Update the output of the module. Set a description of:
```
User data script that should be executed on startup
```

Reply:
done